modules/vstudio/vs2005_dotnetbase.lua

```diff
@@ -278,11 +278,18 @@
 --
 
 	function dotnetbase.debugProps(cfg)
-		if cfg.symbols == p.ON then
-			_p(2,'<DebugSymbols>true</DebugSymbols>')
+		if cfg.symbols == "Full" then
 			_p(2,'<DebugType>full</DebugType>')
+			_p(2,'<DebugSymbols>true</DebugSymbols>')
+		elseif cfg.symbols == p.OFF then
+			_p(2,'<DebugType>none</DebugType>')
+			_p(2,'<DebugSymbols>false</DebugSymbols>')
+		elseif cfg.symbols == "Default" then
+			_p(2,'<DebugType>portable</DebugType>')
+			_p(2,'<DebugSymbols>true</DebugSymbols>')
```

Comment:
I disagree with this. `Default` is supposed to be what the editors default is - not what the editors default is at the time of writing the code.

Reply:
Is there a good way to determine what that is?

Reply:
Don't emit anything and let VS default it? To me, that's what this value is supposed to mean. I did see the comment from @starkos in the linked issue after I commented this though. I personally disagree with the approach of emitting the project templates by default in Premake - it's frustrating having some default baked in that you can't turn off.

Furthermore, this PR will create an inconsistency around what `symbols "Default"` means. Look at what `vcxproj` does, `Default` doesn't emit anything. Look at what the `msc` and `dotnet` tools do, both only support `On`, `Default` doesn't emit anything.

Perhaps a better way to set a "default" value would be using this system instead?
https://github.com/premake/premake-core/blob/6221a49b7eecd7b10b927567efbd9a7545ddf397/src/_premake_init.lua#L1824-L1894

This isn't a hill I'm going to die on or anything, so if you guys feel strongly about emitting `portable` I'm happy for that to happen.

Reply:
> Don't emit anything and let VS default it? To me, that's what this value is supposed to mean.

I've always interpreted it as "match what you'd get by default in a newly created project". For a command line tool that just means don't set the value and let it use its internal default. For IDEs, I've tried to match what you would get if ran `File > New Project...`, since that's what I think most people would consider the "default" behavior. This might differ from not setting the switch at all.

But I do agree with @samsinsane that using "portable" as a blanket fault smells wrong. IMHO, to identify the correct value you'd have to go back and run `File > New Project...` in each major version of the IDE and see what you get. Presumably the logic that is there today reflects what we discovered in the past, so my recommendation would be to see what VS2019 puts in a new project and copy that, making sure not to change the value for any older versions of VS.

My $0.02. Also thanks and h/t @nickclark2016 for digging into this.

Reply:
Works for me.  I'll go back and pull out that so we emit nothing on default and update the tests accordingly.  Appreciate your inputs @samsinsane @starkos.

Reply:
So course of action here is to say if action >= VS2019, then use portable (the default), else continue to use pdbonly? That works for me.  I appreciate the input :)

Reply:
So just to follow up, I threw the old code behind an else block, and used the new code for VS2019 forward.